spec/unit/command/generator_commands/build_cookbook_spec.rb

```diff
@@ -310,7 +310,7 @@ def git!(cmd)
 
         git!("init .")
         git!("add .")
-        git!("commit -m \"initial commit\"")
+        git!("commit --no-gpg-sign -m \"initial commit\"")
```

Comment:
It's really, really annoying to have git prompt for a gpg sig for every single rspec test.